### PR TITLE
feat: KYB-32 - Initiate page routing structure

### PR DESCRIPTION
**Pull request by anusha-kybing**
For time being I have moved `assets` folder inside `src`, I will move this `assets` folder to `public` folder once we eject from `create-react-app`.

### CODE REVIEW

File: src/containers/Home/Home.jsx
Comment (uday-kybing) [inline src/containers/Home/Home.jsx]: Move the css to `src/containers/Home/styles.css`

File: src/components/Logo/Logo.jsx
Comment (uday-kybing) [inline src/components/Logo/Logo.jsx]: Can you please move styles related to this component under same component folder?
And also avoid className and directly use img in `styles.css`
`src/components/Logo/styles.css`
```
img {
  // all App-logo css goes here
}
```

File: src/containers/Header/Header.jsx
Comment (uday-kybing) [inline src/containers/Header/Header.jsx]: Move the css to `src/containers/Header/styles.css`
and use header tag to apply css (delete className)

**Review by uday-kybing — CHANGES_REQUESTED** [review]
Overall looks good

File: src/containers/Home/Home.jsx
Comment (anusha-kybing) [reply src/containers/Home/Home.jsx]: Done.

File: src/containers/Header/Header.jsx
Comment (anusha-kybing) [reply src/containers/Header/Header.jsx]: done

File: src/components/Logo/Logo.jsx
Comment (anusha-kybing) [reply src/components/Logo/Logo.jsx]: done.
Having CSS on img tag will lead to apply CSS to all img tags available on page.

Comment (uday-kybing) [reply src/components/Logo/Logo.jsx]: Okay then styled components will solve this problem I guess. Or we should find a solution to avoid conflicts between css of components.
When we give class names for React components, build should create unique class names for every component so no 2 same class names will conflict. Fundamentally class names can be duplicate but not ids. And css for react components should take full advantage of it.
@amarendhar-kybing I remember you  were working on styled components. Can you please check this. 
We will merge this PR, after renaming `App-logo` to `Logo-img` and all similar

**Review by uday-kybing — APPROVED** [review]
First solid PR. Great work @anusha-kybing

**Comment by amarendhar-kybing** [comment]
@anusha-kybing
I've few questions, i'll write them in the evening, till then please don't merge it.

**Review by amarendhar-kybing — CHANGES_REQUESTED** [review]
@anusha-kybing 
PR looks good to me.
But still i've few questions, i'll write them in the evening, till then please don't merge it.

**Review by amarendhar-kybing — APPROVED** [review]

**Review by amarendhar-kybing — APPROVED** [review]

File: src/index.js
Comment (amarendhar-kybing) [inline src/index.js:22]: we can comment the service-worker code

File: src/containers/Home/Home.css
Comment (amarendhar-kybing) [inline src/containers/Home/Home.css:1]: we should not use class-names starting with capital-letters.

**Comment by amarendhar-kybing** [comment]
- we can just go with ".js" instead of ".jsx" extension.

File: src/index.js
Comment (anusha-kybing) [reply src/index.js:22]: @uday-kybing  told me to keep this

File: src/containers/Home/Home.css
Comment (anusha-kybing) [reply src/containers/Home/Home.css:1]: @uday-kybing  recommended this

**Comment by uday-kybing** [comment]
> * we can just go with ".js" instead of ".jsx" extension.

@amarendhar-kybing Why so?
@anusha-kybing can you please do a quick read about js vs jsx in the long term and share with us to decide

I don't want to block this PR for so long, I don't see any blocker in this PR to stop merging. @amarendhar-kybing You didn't approve this, you had requested for changes. I have requested for re-review. Please check

**Review by amarendhar-kybing — APPROVED** [review]

**Comment by amarendhar-kybing** [comment]
@uday-kybing i `approved` two times after i did `request for changes`.
now also i `approved` one more time, but for some reason, it's not taking it.

**Comment by amarendhar-kybing** [comment]
Screen Shot 2020-07-22 at 7 08 11 AM
Screen Shot 2020-07-22 at 7 08 04 AM

**Comment by uday-kybing** [comment]
@amarendhar-kybing It was because I didn't grand write access to you. 
Only approvals with write access will be counted.

**Comment by amarendhar-kybing** [comment]
@anusha-kybing you can delete the branch, after the merge.

**Pull request merged**
